modular_skyrat/master_files/code/modules/client/preferences/_preference.dm

```diff
@@ -63,6 +63,19 @@
 		target.dna.mutant_bodyparts[relevant_mutant_bodypart] = list(MUTANT_INDEX_NAME = "None", MUTANT_INDEX_COLOR_LIST = list("#FFFFFF", "#FFFFFF", "#FFFFFF"), MUTANT_INDEX_EMISSIVE_LIST = list(FALSE, FALSE, FALSE))
 	target.dna.mutant_bodyparts[relevant_mutant_bodypart][MUTANT_INDEX_EMISSIVE_LIST] = list(sanitize_integer(value[1]), sanitize_integer(value[2]), sanitize_integer(value[3]))
 
+/datum/preference/color/mutant
+	abstract_type = /datum/preference/color/mutant
+
+/datum/preference/color/mutant/apply_to_human(mob/living/carbon/human/target, value)
+	if (type == abstract_type)
+		return ..()
+
+	if(!target.dna.mutant_bodyparts[relevant_mutant_bodypart])
+		target.dna.mutant_bodyparts[relevant_mutant_bodypart] = list(MUTANT_INDEX_NAME = "None", MUTANT_INDEX_COLOR_LIST = list("#FFFFFF", "#FFFFFF", "#FFFFFF"), MUTANT_INDEX_EMISSIVE_LIST = list(FALSE, FALSE, FALSE))
+
+	var/color_value = sanitize_hexcolor(value)
+	target.dna.mutant_bodyparts[relevant_mutant_bodypart][MUTANT_INDEX_COLOR_LIST] = list(color_value, color_value, color_value)
+
```

Comment:
Didn't we already have mutant colors?

Reply:
A triple mutant colour, yes. This is a single colour for use with greyscale things that don't benefit from 2+ colours. It looks cleaner and doesn't burden the user with excess choice that does nothing, a personal gripe of mine with the current menu.